src/chrome/content/rules/Naver.com.xml

```diff
@@ -136,14 +102,10 @@
 		<exclusion pattern="^http://ips\.storefarm\.naver\.com/+(?!css/)" />
 
 
-	<!--	Not secured by server:
-					-->
-	<!--securecookie host="^((admin\.|order\.)?checkout|mileage|sell\.storefarm)\.naver\.com$" name="^WMONID$" /-->
-
 	<securecookie host="^(?:(?:admin\.|order\.)?checkout|sell\.storefarm)\.naver\.com$" name=".+" />
 
 
-	<rule from="^http://(calendar|(?:(?:admin|help|order)\.)?checkout|crossmedia|happybean|help|(?:m\.)?mail[12]?|ssl\.mileage|nid|static\.nid|partners|img\.searchad|sell\.shop|(?:ips|sell)\.storefarm|submit)\.naver\.com/"
+	<rule from="^http://((?:(?:m|section)\.)?blog|calendar|(?:(?:admin|help|order)\.)?checkout|crossmedia|happybean|help|(?:m\.)?mail[12]?|ssl\.mileage|nid|static\.nid|partners|img\.searchad|sell\.shop|(?:ips|sell)\.storefarm|submit)\.naver\.com/"
```

Comment:
We should get rid of all non-capturing groups. Optionally also of the whole wildcard and `not` operators in exclusions.

Reply:
I've removed the `?:` markers in https://github.com/EFForg/https-everywhere/pull/14254/commits/2d18884e075d3a75d7387db3b716d1a832aa134b. I don't want to do a complete overhaul of this ruleset.